libvirt/tests/src/virsh_cmd/pool/virsh_find_storage_pool_sources.py

```
@@ -60,6 +61,30 @@ def run(test, params, env):
+            # Explicitly restart virtstoraged for storage driver ACL tests
+            # The libvirtd.restart() in polkit.setup() doesn't always restart virtstoraged
+            from virttest.staging import service
+            try:
+                virtstoraged = service.Factory.create_service("virtstoraged")
+                logging.info("Restarting virtstoraged to apply polkit rules")
+                virtstoraged.restart()
+                logging.info("virtstoraged restarted successfully")
+            except Exception as svc_err:
+                logging.warning("Failed to restart virtstoraged: %s", svc_err)
+                logging.warning("This may cause ACL test failures for storage operations")
+        except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)